python/dolfinx/mesh.py

```diff
@@ -65,10 +64,159 @@
 ]
 
 
+class Topology:
+    """Topology for a :class:`dolfinx.mesh.Mesh`"""
+
+    _cpp_object: _cpp.mesh.Topology
+
+    def __init__(self, topology: _cpp.mesh.Topology):
+        """Initialize a topology from a C++ topology.
+        Args: The C++ topology object
+        Note:
+            Topology objects should usually be constructed with the
+            :func:`dolfinx.cpp.mesh.create_topology` and not this class initializer.
+        """
+        self._cpp_object = topology
+
+    def cell_name(self) -> str:
+        """String representation of the cell-type of the topology"""
+        return to_string(self._cpp_object.cell_type)
+
+    def connectivity(self, d0: int, d1: int) -> _cpp.graph.AdjacencyList_int32:
+        """Return connectivity from entities of dimension ``d0`` to entities of dimension ``d1``.
+
+        Note:
+            Assumes only one entity type per dimension.
```

Comment:
I don't understand this note.

Reply:
This is taken directly from the C++ docs:
https://github.com/FEniCS/dolfinx/blob/main/cpp/dolfinx/mesh/Topology.h#L106-L130
and I interpret this as:
This connectivity does not take into account prism meshes or mixed meshes, as you would have multiple cell types per dimension. @chrisrichardson Feel free to correct me if I am wrong.

Reply:
From a user's point of view the important point would be to mention that this connectivity is not correct for meshes containing prisms and/or mixed meshes.

Reply:
Let's just remove the comment then. At this stage, anyone using mixed cell types is on their own . . .